File: Amethyst/SiteControl.vb
Public Class siteControl
    Private Sub btnSetName_Click(sender As Object, e As EventArgs) Handles btnSetName.Click
        My.Settings.SiteName = txtbxsetSiteName.Text
    End Sub

    Private Sub trackbarAdIntense_ValueChanged(sender As Object, e As EventArgs)
        My.Settings.AdIntensity = trackbarAdIntense.Value
    End Sub

    Private Sub siteControl_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        txtbxsetSiteName.Text = My.Settings.SiteName
        trackbarAdIntense.Value = My.Settings.AdIntensity
    End Sub

    Private Sub btnSetIntensity_Click(sender As Object, e As EventArgs) Handles btnSetIntensity.Click
        My.Settings.AdIntensity = trackbarAdIntense.Value
    End Sub
End Class

File: Amethyst/SiteControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Amethyst
{
    public partial class SiteControl : UserControl
    {
        public SiteControl()
        {
            InitializeComponent();
        }

        private void btnSetIntensity_Click(object sender, EventArgs e)
        {
            Properties.Settings.Default.AdIntensity = trackbarAdIntense.Value;
        }

        private void btnSetName_Click(object sender, EventArgs e)
        {
            Properties.Settings.Default.SiteName = txtbxsetSiteName.Text;
        }

        private void SiteControl_Load(object sender, EventArgs e)
        {
            txtbxsetSiteName.Text = Properties.Settings.Default.SiteName;
            trackbarAdIntense.Value = Properties.Settings.Default.AdIntensity;
            if (!Properties.Settings.Default.graphicsFriendly)
            {
                btnSetIntensity.FlatStyle = FlatStyle.Flat;
                btnSetName.FlatStyle = FlatStyle.Flat;
            }
        }

        private void trackbarAdIntense_ValueChanged(object sender, EventArgs e)
        {
            Properties.Settings.Default.AdIntensity = trackbarAdIntense.Value;
        }
    }
}
